src/components/issues/Card.tsx

```diff
@@ -69,6 +93,87 @@ const Card: FC<IssueCardProps> = ({ issue }) => {
         }
     };
 
+    const onTaskRequestClick = () => {
+        setSelectedTab(REQUEST_TABS.CREATION_REQUEST);
+    };
+    const onTaskAssignmentClick = () => {
+        setSelectedTab(REQUEST_TABS.TASK_CREATION);
+    };
+    const handleUpdateTask = async (taskData: TaskData, taskId: string) => {
+        try {
+            await updateTask({
+                task: taskData,
+                id: taskId,
+            }).unwrap();
+            toast(SUCCESS, 'Task updated successfully.');
+            setAssignee(taskData.assignee);
+            toggle();
+        } catch (error: any) {
+            toast(ERROR, error.data.message);
+        }
+    };
+    const handleCreateTaskRequest = async (data: any) => {
+        const requestData = {
+            externalIssueUrl: issue.url,
+            userId: userData?.id,
+            requestType: TASK_REQUEST_TYPES.CREATION,
+            proposedStartDate: data.startedOn,
+            proposedDeadline: data.endsOn,
+            description: data.description,
+        };
+        try {
+            const url = TASK_REQUEST_URL;
+            const { requestPromise } = fetch({
+                url,
+                method: 'post',
+                data: requestData,
+            });
+            const response = await requestPromise;
+            setRequestId(response.data.data.id);
+            toast(SUCCESS, 'Task Request created successfully');
+            toggle();
+        } catch (error: any) {
+            if ('response' in error) {
+                toast(ERROR, error.response.data.message);
+                return;
+            }
+            toast(ERROR, error.message);
+        }
+    };
```

Comment:
Why we not using RTK query???

Reply:
Not familiar with it. Will update in next PR. 
https://github.com/Real-Dev-Squad/website-status/issues/963